tests/x11/ghostscript.pm

```diff
@@ -31,31 +31,29 @@ use testapi;
 use utils;
 
 sub run {
-    select_console "x11";
-    x11_start_program('xterm');
+    my $self = shift;
+    $self->select_serial_terminal;
 
-    # become root, disable packagekit and install all needed packages
-    become_root;
+    # disable packagekit and install all needed packages
     quit_packagekit;
     zypper_call "in ghostscript ghostscript-x11";
 
     # special case for gv which is not installed on all flavors
     my $gv_missing = zypper_call("in gv", exitcode => [0, 104]);
-
-    # exit root shell
-    type_string "exit\n";
-
-    my $gs_script = "ghostscript_ps2pdf.sh";
-    my $gs_log    = "ghostscript.log";
-    my $gs_failed = "/tmp/ghostscript_failed";
-    my $reference = "alphabet.pdf";
+    my $gs_script  = "ghostscript_ps2pdf.sh";
+    my $gs_log     = "ghostscript.log";
+    my $gs_failed  = "/tmp/ghostscript_failed";
+    my $reference  = "alphabet.pdf";
 
     # download ghostscript converter script and test if download succeeded
+    # switch to right directory and create files which can be accessed by test user
+    script_run "cd /home/$username";
```

Comment:
I'm pretty sure this will expand to `cd /home/` as `$username` is not defined here.

Reply:
see my test log:

http://10.160.64.152/tests/39/file/autoinst-log.txt

[0m[2021-01-26T09:49:09.999 CET] [debug] tests/x11/ghostscript.pm:50 called testapi::script_run
[2021-01-26T09:49:09.999 CET] [debug] <<< testapi::script_run(cmd="cd /home/bernhard", output="", quiet=undef, timeout=undef)
[2021-01-26T09:49:09.999 CET] [debug] tests/x11/ghostscript.pm:50 called testapi::script_run

your suggestion doesn't work.

Reply:
You were saying?
https://openqa.suse.de/tests/5347984#step/ghostscript/11
https://github.com/mdoucha/os-autoinst-distri-opensuse/blob/a42252183d031a49ddc7a5e9e0432628a25e4d8e/tests/x11/ghostscript.pm#L50

I'd still prefer not to assume that `~$username` points to `/home/$username`.